### PR TITLE
#2369 - Create automatic email notifications for SABC staff (Final PR)

### PR DESCRIPTION
**Pull request by sh16011993**
### As a part of this PR, the following tasks were completed:

- Removed the existing env variable that contains the Ministry email and updated the various pieces of code where it was consumed from the codebase and added it to the release instructions.
- Created the SQL script and added it to the release instructions to have the emails be updated on PROD for the ministry notification to be sent.
- **Code Fix:** Code update to retrieve `institutionLocationName` in `education-program-offering.service.ts`.
- Created story for the e2e tests for the notifications sent to the ministry [Link to the story](https://app.zenhub.com/workspaces/student-information-management-system-5fce9df5aa1b45000e937014/issues/gh/bcgov/sims/3184
)

**SQL:**

```
UPDATE sims.notification_messages AS notification_messages
SET email_contacts = updated_notification_messages.email_contacts
FROM (VALUES (1, '{studentaidbc@gov.bc.ca}'::varchar[]), 
(18, '{studentaidbc@gov.bc.ca}'::varchar[]), 
(19, '{studentaidbc@gov.bc.ca}'::varchar[]), 
(20, '{studentaidbc@gov.bc.ca}'::varchar[]), 
(21, '{studentaidbc@gov.bc.ca}'::varchar[]), 
(22, '{sabc.appealsunit@gov.bc.ca}'::varchar[]), 
(23, '{designat@gov.bc.ca}'::varchar[]), 
(24, '{designat@gov.bc.ca}'::varchar[]), 
(25, '{designat@gov.bc.ca}'::varchar[])) AS updated_notification_messages(id, email_contacts) WHERE updated_notification_messages.id = notification_messages.id;

COMMIT;
```

**Screenshot of release instructions:**

image

### CODE REVIEW

**Review by dheepak-aot — APPROVED** [review]
Looks good. Nice work @sh16011993 

Please add a manual instruction in release notes to remove the github secret `GC_NOTIFY_TO_ADDRESS`

File: sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/ecert-integration/_tests_/ecert-part-time-process-integration.scheduler.e2e-spec.ts [hunk 1]
```diff
@@ -709,7 +708,7 @@ describe(
           {
             initialValue: {
               messagePayload: {
-                email_address: gcNotifyService.ministryToAddress(),
+                email_address: faker.internet.email(),
```
Comment (guru-aot) [inline sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/ecert-integration/_tests_/ecert-part-time-process-integration.scheduler.e2e-spec.ts:711]: 👍

**Review by guru-aot — APPROVED** [review]
Nice work @sh16011993

**Review by andrepestana-aot — APPROVED** [review]
Looks good! Good job!

**Pull request merged**